app/controllers/concerns/school_aggregation.rb

```diff
@@ -25,4 +25,21 @@ def aggregate_school_service
   def aggregate_school
     aggregate_school_service.aggregate_school
   end
+
+  def number_of_solar_readings
+    if @school.solar_pv_tuos_area.present?
+      DataFeeds::SolarPvTuosReading.where(area_id: @school.solar_pv_tuos_area.id).count * 48
+    end
+    0
+  end
+
+  def number_of_weather_readings
+    if @school.weather_station.present?
```

Comment:
If the school has both weather station and dark sky area, should we sum the counts (we'll be using the dark sky data for historic dates)?